src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Noloc/NolocParser.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Microsoft.DocAsCode.MarkdigEngine.Extensions
+{
+    using Markdig.Helpers;
+    using Markdig.Parsers;
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    public class NolocParser : InlineParser
+    {
+        // syntax => ::: noloc text="{content}" :::
+        private const string StartString = "::: noloc text=\"";
+        private const string EndString = "\" :::";
+
+        public NolocParser()
+        {
+            OpeningCharacters = new[] { ':' };
+        }
+
+        public override bool Match(InlineProcessor processor, ref StringSlice slice)
+        {
+            if (!ExtensionsHelper.MatchStart(ref slice, StartString, false))
+            {
+                return false;
+            }
+
+            var text = ExtensionsHelper.TryGetStringBeforeChars(new char[] { '\"', '\n' }, ref slice);
```

Comment:
Please think about this case: 

::: noloc text="dsadsa\
dsadsa" :::

Reply:
added to tests